js/data.js

```diff
@@ -1,8 +1,34 @@
 /* exported data */
 
+// this is the home page
+// data-view="entries"
+
+// this is the forms popup
+// data-view="entry-form"
+
 var data = {
   view: 'entry-form',
   entries: [],
   editing: null,
   nextEntryId: 1
 };
+
+function saveToLocalStorage(event) {
+  var dataModelJSON = JSON.stringify(data);
+  localStorage.setItem('data model', dataModelJSON);
+}
+
+window.addEventListener('submit', saveToLocalStorage);
```

Comment:
This can be deleted, it does the same thing as `saveCurrentView`, but you should be doing it on `beforeunload`, not `submit. I would rename `saveCurrentView` to `saveToLocalStorage` though, it's a more accurate name.